### PR TITLE
[15.0][IMP] stock_picking_product_barcode_report: Allow print from stock_quants

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT48197

ping @carlosdauden @sergio-teruel

### CODE REVIEW

File: stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing_view.xml [hunk 1]
```diff
@@ -84,4 +84,42 @@
         <field name="binding_model_id" ref="stock.model_stock_move_line" />
         <field name="target">new</field>
     </record>
+    <record id="print_picking_label_fishing_report_mini" model="ir.actions.server">
```
Comment (sergio-teruel) [inline stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing_view.xml]: Please set a generic name

**Comment by CarlosRoca13** [comment]
Changed :sweat_smile:

File: stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py [hunk 2]
```diff
@@ -21,7 +21,6 @@ class ProductPrintingQty(models.TransientModel):
     uom_id = fields.Many2one(
         "uom.uom",
         string="Unit of Measure",
-        related="move_line_id.product_uom_id",
```
Comment (carlosdauden) [inline stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py:24]: Is this change contemplated for the rest of the wizard calls or would it be advisable to make this field compute readonly=False?

Comment (carlosdauden) [inline stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py:24]: Is this change contemplated for the other wizard calls?

Comment (CarlosRoca13) [reply stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py:24]: Yes, it is filled from move_line_ids in this method: https://github.com/OCA/stock-logistics-barcode/blob/1b8a23f172608a4afc2e803e7e312a1901c96ca8/stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing.py#L135

IDK why it was related if it was filled there :sweat_smile:

**Review by carlosdauden — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

File: stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing_view.xml [hunk 3]
```diff
@@ -84,4 +84,42 @@
...
         <field name="target">new</field>
     </record>
+    <record id="act_stock_barcode_selection_printing_quant" model="ir.actions.server">
+        <field name="name">Print barcode labels</field>
+        <field name="model_id" ref="stock.model_stock_quant" />
+        <field name="binding_model_id" ref="stock.model_stock_quant" />
+        <field name="binding_type">report</field>
+        <field name="state">code</field>
+        <field name="code">
+lines = []
```
Comment (pedrobaeza) [inline stock_picking_product_barcode_report/wizard/stock_barcode_selection_printing_view.xml]: This bunch of code is very likely to fail without tests. It's even not very recommended to add it this way. Can you add at least a test or convert it to other technique?

**Comment by CarlosRoca13** [comment]
@pedrobaeza changed and test added

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**